src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -969,6 +972,8 @@ class module_sram : public module_impl
     if (m_ctrlpkt_bo) {
       m_column_bo_address.push_back({ m_ctrlpkt_bo.address(), m_ctrlpkt_bo.size() }); // NOLINT
     }
+    m_column_bo_address.push_back({ m_preempt_save_bo.address(), m_preempt_save_bo.size() }); // NOLINT
```

Comment:
I am hoping firmware have a way to figure out whether there is control packet or not.
Hopefully, firmware will not think m_preempt_save_bo is m_ctrlpkt_bo